packages/plugin-commands-doctor/src/doctor.ts

```diff
@@ -0,0 +1,60 @@
+import renderHelp from 'render-help'
+import { docsUrl } from '@pnpm/cli-utils'
+import { logger } from '@pnpm/logger'
+import loadNpmConf from '@pnpm/npm-conf'
+
+export const rcOptionsTypes = cliOptionsTypes
+
+export function cliOptionsTypes () {
+  return {
+    config: Boolean,
+  }
+}
+
+export const shorthands = {
+  C: '--config',
+}
+
+export const commandNames = ['doctor']
+
+export function help () {
+  return renderHelp({
+    description: 'Checks for known common issues.',
+    descriptionLists: [
+      {
+        title: 'Options',
+        list: [
+          {
+            description: 'Check the global config',
+            name: '--config',
+            shortAlias: '-C',
+          },
+        ],
+      },
+    ],
+    url: docsUrl('doctor'),
+    usages: ['pnpm doctor [options]'],
+  })
+}
+
+export async function handler (
+  opts: {
+    config: boolean
+  }) {
+  if (opts.config) {
+    const { failedToLoadBuiltInConfig } = loadNpmConf()
```

Comment:
just update the lib in `@pnpm/config` and add `failedToLoadBuiltInConfig` to the `pnpmConfig` object. It will be then passed to the opts.

Reply:
Thanks~